portal/src/main/resources/applicationContext-security.xml

```diff
@@ -473,6 +472,58 @@
         </b:bean>
     </b:beans>
 
+    <!-- begin ldap beans -->
+    <b:beans profile="ldap">
+        <b:bean
+            class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+            <b:property name="staticMethod"
+                value="org.mskcc.cbio.portal.util.SpringUtil.setAccessControl" />
+            <b:property name="arguments">
+                <b:list>
+                    <b:ref bean="accessControl" />
+                </b:list>
+            </b:property>
+        </b:bean>
+
+        <http use-expressions="true" auto-config="true">
+            <form-login login-page="/login.jsp" login-processing-url="/j_spring_security_check" default-target-url="/index.do" />
+            <logout     logout-success-url="/login.jsp?logout_success=true" delete-cookies="JSESSIONID" />
+            <intercept-url pattern="/auth/*" access="permitAll"/>
+            <intercept-url pattern="/favicon.ico" access="permitAll"/>
+            <intercept-url pattern="/login.jsp*" access="permitAll"/>
+            <intercept-url pattern="/webservice.do*" access="isAuthenticated() or hasIpAddress('127.0.0.1')"/>
+            <intercept-url pattern="/**" access="isAuthenticated()"/>
+
+            <!-- to enable access from matlab, r, python, etc clients -->
+            <session-management session-fixation-protection="none" />
+        </http>
+
+        <ldap-server id="ldapServer" url="${ldap.url}"
+            manager-dn="${ldap.manager.dn}" manager-password="${ldap.manager.password}" />
+
+        <!-- Loading full name and email from AD after authentication -->
+        <b:bean id="userDetailsService"
+            class="org.mskcc.cbio.portal.authentication.ldap.LDAPUserDetailsContextMapper">
```

Comment:
@holtgrewe thanks! I'll let you fix it ;)  I believe I tested the one from #1076 today and it worked fine (I added some comments there). Maybe it is the same as #1072, I didn't check.